src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -314,7 +314,8 @@ private void processFiles(List<File> files) throws CheckstyleException {
                 }
 
                 // We need to catch all errors to put a reason failure (file name) in error
-                throw new Error("Error was thrown while processing " + file.getPath(), error);
+                throw new Error("Error was thrown while processing "
+                        + file.getAbsolutePath(), error);
```

Comment:
Can we keep getPath ?
Looks like just extra assert is required in tests

Reply:
I think yes can you please guide in which sceanrio we are throwing this error

Reply:
I think test is failing at https://dev.azure.com/romanivanovjr/romanivanovjr/_build/results?buildId=17141&view=logs&j=c902ebb4-c9f8-5f09-4e17-ff78fbbc842e&t=9ca98c81-ff64-58f0-9d03-a23ac1c4a111&l=1322

We have full coverage so you can easily find it by message or change type of exception and some test will fail

Reply:
restored changes